### DISCUSSION TITLE
Does unstable_revalidate work only for Next.js?

### DISCUSSION THREAD

**Discussion by kmcaloon** [vercel/community#661] [Help] [answered]
So on-demand ISR is a huge addition to Next.js, but I was wondering if similar functionality is available for apps build with other frameworks and deployed according to the Build Output API. For instance, say we have route with a prerender function cached with a `stale-while-revalidate` `Cache-Control` header. Would we be able to revalidate the cache for that route with a call to a an edge function that uses `unstable_revalidate`?

**Comment by leerob** [comment]
Hey! Currently, it only works for Next.js, but we are planning to add support for On-Demand ISR to the Build Output API to enable any framework to use this.

**Comment by kmcaloon** [comment]
Thanks @leerob, I'll keep an eye out. Love everything you guys are doing over there 👍

**Comment by leerob** [comment, marked as answer]
Update: This is now possible! https://github.com/vercel/examples/pull/306
